modules/core/src/test/java/org/apache/ignite/internal/processors/metastorage/persistence/DistributedMetaStoragePersistentTest.java

```diff
@@ -84,6 +86,32 @@ public void testRestart() throws Exception {
         assertEquals("value", ignite.context().distributedMetastorage().read("key"));
     }
 
+    /**
+     * @throws Exception If failed.
+     */
+    @Test
+    public void testDeactivateWriteActivate() throws Exception {
```

Comment:
Will it be possible to add a comment, what does this test check from its name and contents is a little unclear?
At first glance it looks quite simple, but maybe it checks something more non-obvious?

Reply:
I will add a comment, but it is that simple actually. I realized that I forgot this scenario in an initial set of simple tests